client/components/category-pill-navigation/index.tsx

```diff
@@ -70,6 +74,33 @@ export const CategoryPillNavigation = ( {
 		} );
 	}, [ selectedCategoryId ] );
 
+	if ( isMobile() ) {
+		return (
+			<div className="category-pill-navigation">
+				<FormSelect
```

Comment:
Did you consider `SelectDropdown` and `SelectDropdown.Item` from `@automattic/components`? It'd be nice to avoid the default browser select menu if we can, IMO.

Reply:
i1 should have native dropdown, and later we will implement custom when the design is ready. Answer from Noemi p1711535798739269/1711409617.975969-slack-C06ELHR6L9J

Reply:
The way I understand Noemi's answer is that the design team has a custom dropdown in mind, but if the native dropdown is faster to implement, we can use that for the first iteration. If you've looked into `SelectDropdown` and think it's too time-intensive to implement right now, we could stick with `FormSelect` for now and take another look next week. Is that the case?

Reply:
>it's too time-intensive to implement right now

Totally no. I don't see any difference between temporary solutions. Only tiny thing:
1) Native <select> is definitelly simple thing, users are used to it and it works definitely always good in any environment / place.
2) SelectDropdown is a custom solution, so maybe designers can consider it as a not good UI for teh current usecase. It's just assumption, of using custom solutions. But likely hood is quite low so I totally don't mind of using SelectDropdown here.